diffusion/models/models.py

```diff
@@ -33,7 +34,10 @@
 def stable_diffusion_2(
     model_name: str = 'stabilityai/stable-diffusion-2-base',
     pretrained: bool = True,
+    autoencoder_path: Optional[str] = None,
+    autoencoder_local_path: str = '/tmp/autoencoder_weights.pt',
     prediction_type: str = 'epsilon',
+    latent_scale: Optional[float] = None,
```

Comment:
Why not?
```suggestion
    latent_scale: Optional[float] = 0.18215,
```

Reply:
Ah, I see.. to have an option to use `latent_statistics` value. But should this be made an explicit argument?  At the expense of another argument

Reply:
Would you be ok with leaving it as `Optional` but having the default be the `0.18215` value? I think that would allow a user to explicitly set `latent_scale = None` to attempt to pick up the `latent_statistics` value.

Reply:
I think that would be better.

I still don't like the extra `if latent_scale is None: latent_scale = 0.18215`, but I don't think there is a way to avoid it without adding another argument so it's okay.

Reply:
Hmm, if you like, the `if latent_scale is None: latent_scale = 0.18215` code could be removed in `model.py` since the `StableDiffusion` class also tries to default to the correct value in the event that `latent_scale` is `None` [here](https://github.com/mosaicml/diffusion/blob/c398359ea28ed0ffd5d7a36723de3b4cb16265fc/diffusion/models/stable_diffusion.py#L117). Though this is just the same ugliness in a different place. 

I'd really like to avoid having to introduce an extra argument for this in any case. What about allowing `latent_scale` to take on a value of `'auto'` to try use the value logged in the autoencoder checkpoint, and leaving the default value the correct number for the base model?

Reply:
I was thinking that in the case of `auto`, we'd only allow `latent_scale` to be a `float` (defaulting to the correct value) or a `str`, and throw an error if it's `auto` without an autoencoder checkpoint to load stats from. So, the check for `None` could be removed. I think a setting for `latent_statistics` would work similarly. Definitely open to making it more specific. I'd rather get this right the first time than having something else to clean up later.

Reply:
To me, `'latent_statistics'` is a lot clearer than `'auto'`

Reply:
Ok, I made this change!